src/DssSpell.sol

```diff
@@ -59,106 +44,41 @@ contract SpellAction {
-        //MedianAbstract(OsmAbstract(PIP_GUSD).src()).kiss(PIP_GUSD);
-        // Whitelist Spotter to read the Osm data (only necessary if it is the first time the token is being added to an ilk)
-        // !!!!!!!! Only if PIP_GUSD = Osm or PIP_GUSD = Median and hasn't been already whitelisted due a previous deployed ilk
-        //OsmAbstract(PIP_GUSD).kiss(MCD_SPOT);
-        // Whitelist End to read the Osm data (only necessary if it is the first time the token is being added to an ilk)
-        // !!!!!!!! Only if PIP_GUSD = Osm or PIP_GUSD = Median and hasn't been already whitelisted due a previous deployed ilk
-        //OsmAbstract(PIP_GUSD).kiss(MCD_END);
-        // Set GUSD Osm in the OsmMom for new ilk
-        // !!!!!!!! Only if PIP_GUSD = Osm
-        //OsmMomAbstract(OSM_MOM).setOsm(ilk, PIP_GUSD);
+        address MCD_JUG      = CHANGELOG.getAddress("MCD_JUG");
+        address MCD_VAT      = CHANGELOG.getAddress("MCD_VAT");
 
         // Set the global debt ceiling
-        VatAbstract(MCD_VAT).file("Line", 1_468_750_000 * RAD);
-        // Set the GUSD-A debt ceiling
-        VatAbstract(MCD_VAT).file(ilk, "line", 5 * MILLION * RAD);
-        // Set the GUSD-A dust
-        VatAbstract(MCD_VAT).file(ilk, "dust", 100 * RAD);
-        // Set the Lot size
-        CatAbstract(MCD_CAT).file(ilk, "dunk", 50 * THOUSAND * RAD);
-        // Set the GUSD-A liquidation penalty (e.g. 13% => X = 113)
-        CatAbstract(MCD_CAT).file(ilk, "chop", 113 * WAD / 100);
-        // Set the GUSD-A stability fee (e.g. 1% = 1000000000315522921573372069)
-        JugAbstract(MCD_JUG).file(ilk, "duty", FOUR_PERCENT_RATE);
-        // Set the GUSD-A percentage between bids (e.g. 3% => X = 103)
-        FlipAbstract(MCD_FLIP_GUSD_A).file("beg", 103 * WAD / 100);
-        // Set the GUSD-A time max time between bids
-        FlipAbstract(MCD_FLIP_GUSD_A).file("ttl", 6 hours);
-        // Set the GUSD-A max auction duration to
-        FlipAbstract(MCD_FLIP_GUSD_A).file("tau", 6 hours);
-        // Set the GUSD-A min collateralization ratio (e.g. 150% => X = 150)
-        SpotAbstract(MCD_SPOT).file(ilk, "mat", 101 * RAY / 100);
-
-        // Update GUSD-A spot value in Vat
-        SpotAbstract(MCD_SPOT).poke(ilk);
-
-        // Add new ilk to the IlkRegistry
-        IlkRegistryAbstract(ILK_REGISTRY).add(MCD_JOIN_GUSD_A);
+        // 1,468,750,000 (DC after 11-13 spell executes) + 13 (YFI-A increase)
+        VatAbstract(MCD_VAT).file("Line", 1_481_750_000 * RAD);
+
+        // Increase YFI debt ceiling to 20MM
+        VatAbstract(MCD_VAT).file("YFI-A", "line", 20 * MILLION * RAD);
+
+        // Set dust to 500 DAI
+        bytes32[] memory ilks = IlkRegistryAbstract(ILK_REGISTRY).list();
+        for (uint256 i = 0; i < ilks.length; i++) {
+            VatAbstract(MCD_VAT).file(ilks[i], "dust", 500 * RAD);
+        }
+
+        // Set stablecoin fees to ????
```

Comment:
We can get rid of these now. Just announced.

"a stability fee change for stablecoins will not move forward to the executive this week."

https://forum.makerdao.com/t/update-2-poll-outcomes-2020-11-19/5235/10

Reply:
Oops just saw this haha.

Reply:
Done.